toolset/setup/linux/bash_functions.sh

```diff
@@ -132,7 +131,8 @@ fw_depends() {
       set -x
       . $FWROOT/toolset/setup/linux/frameworks/${depend}.sh
     else
-      echo WARN: No installer found for $depend
+      echo WARN: No installer found for $depend, attempting to install with 'apt-get'...
+      sudo apt-get install -o Dpkg::Options::="--force-confold" --force-yes ${depend}
```

Comment:
There's another `Dpkg::Options` flag we should use in addition to the one here: `Dpkg::Options::="--force-confdef"`